website/pages/blog/accounts.js-1.0-rc.mdx

```diff
@@ -0,0 +1,217 @@
+
+OAuth authentication, while working, surely deserves some love. While REST endpoints for OAuth should be functional there are no mutations nor resolvers yet, meaning that you can't use it with the GraphQL transport. I'd also like to go through the OAuth code and write some examples. `@accounts/oauth-instagram` in particular still relies upon the deprecated `request` package and should be updated. Other popular OAuth providers like Facebook still need to be added (but PRs exist).
+
+Before 1.0 stable gets released I plan to at least get the existing providers in a better shape, together with examples and the relevant GraphQL schema.
+
+## Post 1.0
+
+I'd like to create a new `@accounts/phone` authentication service which lets you authenticate via SMS OTPs. That would be especially useful in react-native applications where you could automatically read the sms and automate the authentication process.
+
+Currently Accounts.js bundles CommonJS code. While CommonJS can be imported in both CommonJS and ESM applications that would rule out Deno/Bun support. For the same reason we cannot use a wrapper either: while that would allow to use ESM imports it wouldn't be real ESM and thus won't be compatible. The remaning alternatives are pure ESM and dual packages. While several library authors opted for the former because of [dual package hazard](https://nodejs.org/api/packages.html#packages_dual_package_hazard) concerns I weight the benefits differently. While dual package hazards are real the whole GraphQL echosystem relies on dual packages thus using `instanceof` is already an hazard. These authors suggest to use `async imports` to import their pure ESM libraries in commonjs projects, which exposes everyone to the very same hazard (not even considering the fact that this is only viable in async contexts). That basically goes against the reason why they decided to bundle pure esm in the first place. I personally think that it's still too early to target pure esm and dual package is the lesser evil so I'm leaning towards that for future releases, but I'm ready to change my mind if you provide me enough arguments.
+
+I'd also like to implement some form of accounts linking, where users could link their existing account with a different authentication service (for example password and oauth).
+
+I would like to a extend Multi Factor Authentication outside of the password service, baking it into the core of accounts.js so that any authentication service can take advantage of it.
+
+If future major versions of Accounts.js will introduce breaking changes to the database structure I would like to provide migrations directly via the `@accounts/mikro-orm` package.
+
+Last but not the least I would like to bake in cookies authentication. Not only that would fare better against XSS, it would also allow server side rendering and thus enable the usage of frameworks like Next.js. Alternative storage methods would remain available for those using native applications.
```

Comment:
```suggestion
Last but not least, I would like to bake in cookies authentication. Not only would that fare better against XSS, but it would also allow server-side rendering and thus enable the usage of frameworks like Next.js. Alternative storage methods would remain available for those using native applications.
```